etc/module.xml

```diff
@@ -1,4 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
-    <module name="Omise_Payment" setup_version="2.3.0" />
+    <module name="Omise_Payment" setup_version="2.3.0">
+    	<sequence>
+    		<module name="Magento_Sales"/>
+    		<module name="Magento_Payment"/>
+    		<module name="Magento_Checkout"/>
```

Comment:
@jonrandy just tab-to-space thing here 😄

Reply:
![image](https://user-images.githubusercontent.com/1510194/40899134-894edade-67ef-11e8-94e7-e897659070d4.png)

Reply:
I know 😜